applications/API/views.py

```
@@ -0,0 +1,655 @@
+import django.utils.decorators
+import django.views.decorators.cache
```

Ruff (0.12.2)

`django.views.decorators.cache` imported but unused

Remove unused import: `django.views.decorators.cache`

(F401)